blender/LilySurfaceScrapper/Scrappers/AbstractScrapper.py

```diff
@@ -28,8 +28,13 @@
 try:
     from lxml import etree
 except ImportError:
-    print("No system-wide installation of lxml found. Falling back to local version.")
-    sys.path.append(os.path.join(os.path.dirname(os.path.dirname(os.path.realpath(__file__))), "site-packages"))
+    print("No system-wide installation of lxml found. Installing a local version...")
+    import subprocess
+    binary_path_python = sys.executable
+    if binary_path_python.endswith(("blender", "blender.exe")):
+        import bpy
```

Comment:
Still not that familiar with python: Can we just import the function here & would that be more efficient?

Reply:
`from bpy import stuf` to import only stuf. Reducing the number of imported symbols makes the global function name look up faster, but I believe it is totally marginal here, I wouldn't bother! I was not aware of the existence of  `bpy.app.binary_path_python`, good to know it now. :)